pkg/pillar/hypervisor/qmp.go

```diff
@@ -0,0 +1,63 @@
+package hypervisor
+
+import (
+	"encoding/json"
+	"github.com/digitalocean/go-qemu/qmp"
+	"time"
+)
+
+// this package implements subset of
+//     https://qemu.weilnetz.de/doc/qemu-qmp-ref.html
+
+const sockTimeout = 10 * time.Second
+
+func execRawCmd(socket, cmd string) ([]byte, error) {
+	monitor, err := qmp.NewSocketMonitor("unix", socket, sockTimeout)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := monitor.Connect(); err != nil {
+		return nil, err
+	}
+	defer monitor.Disconnect()
+
+	return monitor.Run([]byte(cmd))
+}
+
+func execCont(socket string) error {
```

Comment:
I think it would be more clear if this was named execContinue; I first thought it was an abreviation for execContainer.

Reply:
Fixed